src/js/decorations/paragraph.js

```diff
@@ -5,9 +5,17 @@ export default (
 ) => (from, to, doc, decos) => {
   let newDecos = decos;
   doc.nodesBetween(from, to, (node, pos) => {
-    newDecos = predicate(node)
-      ? newDecos.add(doc, [createDeco(pos + node.nodeSize - 1, 'par')])
-      : newDecos;
+    if (predicate(node)) {
+      const decoPos = pos + node.nodeSize - 1;
+      const oldDecos = newDecos.find(
+        decoPos,
+        decoPos,
+        spec => spec.key === 'par'
```

Comment:
Maybe just break this out to a constant, can see no reason why it would change but the bug could easily reappear silently if only one of these were to change.

Reply:
Good idea. I will.

Reply:
Done.